src/views/Scenario/Scenario.js

```diff
@@ -276,7 +276,8 @@ const Scenario = (props) => {
                   <Grid item xs={7} className={classes.scenarioMetadata}>
                     {scenarioValidationArea}
                     <Typography className={classes.scenarioRunTemplateLabel}>
-                      {t('views.scenario.text.scenariotype')}: {currentScenario.data.runTemplateName}
+                      <span>{t('views.scenario.text.scenariotype')}: </span>
+                      <span data-cy="run-template-name">{currentScenario.data.runTemplateName}</span>
```

Comment:
Note sure if using `span`s in a `Typography` is a good practice, but you could set the `data-cy` on the `Typography` : the call to `should('have.text', ...)` should detect the pattern even if it is only a part of the whole text.